tests/meta/demultiplex/test_validation.py

```diff
@@ -109,18 +109,18 @@ def test_is_sample_id_in_directory_name_with_valid_directory():
     directory = Path("Sample_123")
 
     # WHEN checking if the sample id is in the directory name
-    result = is_sample_id_in_directory_name(directory)
+    result = is_sample_id_in_directory_name(directory=directory, sample_internal_id="Sample_123")
 
     # THEN the result should be True
     assert result is True
```

Comment:
Does this test work? Seems to me as if the result should be False since we check "if "Sample_Sample_123" in "Sample_123""?

Reply:
Good catch!

Reply:
Changed the test to not be as confusing. The validation function just checks if the sample id is present in the directory name